src/main/java/io/xpring/xrpl/Utils.java

```diff
@@ -86,7 +88,11 @@ public static String encodeXAddress(ClassicAddress classicAddress) {
    */
   @SuppressWarnings("checkstyle:ParameterName")
   public static ClassicAddress decodeXAddress(String xAddress) {
-    return javaScriptUtils.decodeXAddress(xAddress);
+    try {
+      return XAddressCodec.decode(xAddress);
```

Comment:
Just noting from my comment about `xaddress` above that we use `xAddress` here. Whichever way you choose we should try to be consistent.